sklearn/decomposition/incremental_pca.py

```diff
@@ -21,7 +21,7 @@ class IncrementalPCA(_BasePCA):
     but not scaled for each feature before applying the SVD.
 
     Depending on the size of the input data, this algorithm can be much more
-    memory efficient than a PCA.
+    memory efficient than a PCA, and allows sparse input.
```

Comment:
I think you should note somewhere that each batch will be densified, and so batch size and number of features can affect computation substantially.

Reply:
Do you mean computation with regards to memory use? Or the computational cost of conversion? The latter should be negligible compared to SVD.